js/reference.js

```diff
@@ -16,6 +16,17 @@
 
 var ERMrest = (function(module) {
 
+    module._contexts = Object.freeze({
```

Comment:
I am still unsure whether we should use `Object.freeze` due to its browser compatibility issue. Can't we just use a simple javascript getter for this ? 

[object.defineproperty](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty)  will work in most browsers apart from IE8.